db/db_test.go

```diff
@@ -0,0 +1,25 @@
+package db
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBuildDataSourceName(t *testing.T) {
+	assert := assert.New(t)
+
+	dataSourceName := buildDataSourceName()
+
+	assert.Equal(dataSourceName,
```

Comment:
hm, can we actually use the config here to test this? what if we change something in config, then we will have to change it here as well